erp-server/src/main/java/com/ttt/erp/controller/SignatureController.java

```diff
@@ -22,11 +24,23 @@
     @Autowired
     private UserAccountService userService;
 
+    @Autowired 
+    private UserAccountRepository userRepo;
+
+    @Autowired 
+    private AwsS3ClientService s3Client;
+
     @GetMapping
-    public Optional<String> findSignature(Principal principal) {
+    public Optional<String> findSignatureName(Principal principal) {
         return service.getSignatureForUsername(principal.getName());
     }
 
+    @GetMapping("/file")
+    public void findSignatureFileData(@CookieValue(value = "user") String actingUser) {
+        String fName = this.userRepo.findByUsername(actingUser).getSignature();
+        System.out.println(this.service.getSignatureBase64(fName));
+    }
```

Comment:
I don't think we're even using this controller anywhere 🤔. Have mostly just been using it for testing.